src/query.rs

```diff
@@ -238,16 +238,26 @@ mod tests {
             .expect("failed to load expected outputs");
         let mut expected_results: Vec<BTreeMap<String, FieldValue>> =
             ron::from_str(&expected_result_text)
-                .expect("could not parse expected outputs as ron format");
-
-        let adapter = VersionedRustdocAdapter::new(&current, Some(&baseline))
-            .expect("could not create adapter");
-        let results_iter = adapter
-            .run_query(&semver_query.query, semver_query.arguments.clone())
-            .unwrap();
-
-        let mut actual_results: Vec<BTreeMap<_, _>> = results_iter
-            .map(|res| res.into_iter().map(|(k, v)| (k.to_string(), v)).collect())
+                .expect("Could not parse expected outputs as ron format.");
```

Comment:
```suggestion
                .expect("could not parse expected outputs as ron format");
```

Reply:
Probably best not to change the style of `.expect()` calls, and instead match the existing one.